package.json

```diff
@@ -3,18 +3,20 @@
   "version": "0.1.0",
   "private": true,
   "devDependencies": {
-    "@wdio/cli": "^7.16.14",
-    "@wdio/local-runner": "^7.16.15",
-    "@wdio/mocha-framework": "^7.16.15",
-    "@wdio/spec-reporter": "^7.16.14",
+    "@wdio/cli": "^7.19.7",
+    "@wdio/local-runner": "^7.19.7",
+    "@wdio/mocha-framework": "^7.19.7",
+    "@wdio/spec-reporter": "^7.19.7",
     "@wdio/static-server-service": "^7.16.14",
     "autoprefixer": "^10.4.2",
+    "chromedriver": "^101.0.0",
     "cpx-fixed": "^1.6.0",
     "npm-run-all": "^4.1.5",
     "prettier": "^2.5.1",
     "prettier-plugin-tailwindcss": "^0.1.7",
     "simple-autoreload-server": "^0.2.10",
-    "tailwindcss": "^3.0.22"
+    "tailwindcss": "^3.0.22",
+    "wdio-chromedriver-service": "^7.3.2"
```

Comment:
Install chromedriver using `npm install -g chromedriver` instead. Remove it from the package.json and package-lock.json.